swap_meet/clothing.py

```diff
@@ -1,2 +1,27 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, condition = 0, fabric=None):
+        super().__init__(id,condition)
+        if fabric:
+            self.fabric = fabric
+        else:
+            self.fabric = "Unknown"
+
+#inherit get_category function from Item class, return class name   
+    def get_category(self):
```

Comment:
Since there was no additional functionality in the child's method, you didn't have to reimplement this here. It should automatically call the parent class's version if there isn't anything defined in the child :)